docu/files/HowToGuideBuilderEditor.html.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Implementation guide for builder editor module V3 '
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: 5
+---
+
+This brief how-to guide uses version 3 of the builder_editors module (**builder_editor_v3.js**) to outline the necessary implementation steps. This page serves as an addendum to the existing documentation for versions 1 and 2.
+
+
+* Include the builder module: https://builder.yaas.io/public/js/builder_editors_v3.js<br>
+* Add 'builder_editors' module to your angular module.<br>
+* Prepare the configuration: <br>
+
+```
+ $scope.configuration = {
+    'en': {'default': true, 'required': true},
+    'de': {'default': false, 'required': false}
+ };
+
+```
+* Next, specify the data that you would like to display in the element.  Here is an example:             
+
+```
+   $scope.localizedData = {
+   'en': 'english text',
+   'de': 'german text',
+   };
+   
+```
+* Modify your html to include the editor with its corresponding attributes.
+
+Example:
+
+```
+<div class="panel-body">
+ <h3>Single Line</h3>
```

Comment:
The H1 headings automatically render in Title Case, but H3 and smaller headings should be in Sentence case, as in: Single line